codalab/codalabtools/compute/requirements.txt

```diff
@@ -0,0 +1,9 @@
+psutil==4.0.0
+PyYAML==3.10
+
+
+# Added for python sdk
+git+https://github.com/FlavioAlexander/azure-sdk-for-python.git@legacy
```

Comment:
I think once we merge in develop we can remove this, compute worker doesn't need Azure any more.